src/issue-strategies/story.js

```diff
@@ -8,7 +9,11 @@ function areParentsValid(baseIssueKey, parentIssue, jiraClientAPI) {
       } else if(content.fields.issuetype.name === 'MT' ||  // TODO: Is this one old??
                 content.fields.issuetype.name === 'Maintenance Task' ||
                 content.fields.issuetype.name === 'Bug' ||
-                content.fields.issuetype.name === 'Initiative') {
+                content.fields.issuetype.name === 'Initiative' ||
+                content.fields.issuetype.name === 'Dispatcher'){
+        if((content.fields.issuetype.name === 'Maintenance Task' || content.fields.issuetype.name === 'Bug') && content.fields.issuelinks){
+          return bugMtStrat.apply(content, jiraClientAPI);
+        }
```

Comment:
This is rough. I don't think the `content.fiels.issuelinks` check is necessary. If we're going to have maintenance tasks and bugs do something different we may as well define those in a completely separate if block. Since you're already here we can nix that `else` statement.